shuffle/cli/api.md

```diff
@@ -0,0 +1,143 @@
+
+`shuffle deploy --project-path <project_path> --network <network_name>` |
+
+`shuffle deploy --project-path ~/Desktop/TestCoin --network trove_testnet`
+
+Publishes the main move package onto specified network. Note: the network name that is passed in must exist in the Networks.toml file.
+
+## shuffle console
+
+**Usage:**
+
+`shuffle console --project-path <project_path>` | `shuffle console --project-path ~/Desktop/TestCoin`
+
+Enters typescript REPL for onchain inspection of deployed project.
+
+`shuffle console --project-path <project_path> --network <network_name>` |
+
+`shuffle console --project-path ~/Desktop/TestCoin --network trove_testnet`
+
+Enters REPL for onchain inspection on specified network. Note: the network name that is passed in must exist in the Networks.toml file.
+
+`shuffle console --project-path <project_path> --key-path <private_key_path> --address <account_address>` |
+
+`shuffle console --project-path ~/Desktop/TestCoin --key-path ~/.shuffle/networks/localhost/accounts/latest/dev.key --address 0x24163AFCC6E33B0A9473852E18327FA9`
+
+Enters repl for inspection on certain key_path and address. Note: when using the key_path and address flags, they both must be passed in.
+
+## shuffle build
+
+**Usage:**
+
+`shuffle build --project-path <project_path>` | `shuffle build --project-path ~/Desktop/TestCoin`
+
+Compiles the move package in the user's project folder and generates typescript files.
+
+## shuffle test
+
+**Usage:**
+
+`shuffle test e2e --project-path <project_path>` | `shuffle test e2e --project-path ~/Desktop/TestCoin`
+
+Runs end-to-end test in shuffle with project folder on localhost
+
+`shuffle test e2e --project-path <project_path> --network <network_name>` |
+
+`shuffle test e2e --project-path ~/Desktop/TestCoin --network localhost`
+
+Runs end-to-end test in shuffle with project folder on specific network.
+Note: the network name that is passed in must exist in the Networks.toml file.
+
+`shuffle test unit --project-path <project_path>` |
+
+`shuffle test unit --project-path ~/Desktop/TestCoin`
+
+Runs move unit tests in project folder
+
+`shuffle test all --project-path <project_path> --network <network_name>` |
+
+`shuffle test all --project-path ~/Desktop/TestCoin --network trove_testnet`
+
+Runs both move unit tests in project folder and end-to-end test in shuffle with project folder on specific network.
+Note: the network name that is passed in must exist in the Networks.toml file.
+
+## shuffle transactions
+
+**Usage:**
+
+`shuffle transactions`
+
+Captures last 10 transactions from the account on the localhost network in pretty formatting.
+
+`shuffle transactions --raw`
+
+Captures last 10 transactions from the account on the localhost network without pretty formatting.
+
+`shuffle transactions --tail`
+
+Captures last 10 transactions from the account on the localhost network in pretty formatting and blocks/continuously polls for incoming transactions.
```

Comment:
it might be good to use some sort of header separator and then you can use smaller headers as follows, so that these might be slightly more visible.

also `--tail` by itself typically won't result in an automatic follow without `--follow` or maybe or the such.

Reply:
Yes. this is a typo. should be -t